packages/synapse-interface/components/_Transaction/helpers/getEstimatedTimeStatus.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Provide additional data around elapsed time from an initial start time
+ *
+ * @param currentTime in s, unix
+ * @param initialTime in s, unix
+ * @param estimatedTime in s, unix
+ */
+const oneMinuteInSeconds = 60
+
+export const getEstimatedTimeStatus = (
+  currentTime: number,
+  initialTime: number,
+  estimatedTime: number
+) => {
+  const elapsedTime = currentTime - initialTime
+  const remainingTime = estimatedTime - elapsedTime
+
+  const remainingTimeInMinutes = Math.ceil(remainingTime / oneMinuteInSeconds)
+  const elapsedTimeInMinutes = Math.ceil(elapsedTime / oneMinuteInSeconds)
+
+  const isEstimatedTimeReached = remainingTime < 0
+  const isStartCheckingTimeReached = remainingTime < oneMinuteInSeconds
+
+  return {
+    elapsedTime,
+    elapsedTimeInMinutes,
```

Comment:
Similar comment about time before. Additionally `Math.ceil` truncates a value (presumably to show buffer), but we don't want that if we're using it to make comparisons. Would remove all the `inMinutes` fields from this hook) and just use seconds and convert to minutes at the end of where they're used.